lib/auth/usage.go

```diff
@@ -18,20 +18,15 @@ import (
 	"context"
 	"time"
 
-	"github.com/gravitational/teleport/api/client/proto"
 	apidefaults "github.com/gravitational/teleport/api/defaults"
 	"github.com/gravitational/teleport/api/types"
 	apievents "github.com/gravitational/teleport/api/types/events"
 	"github.com/gravitational/teleport/lib/events"
-	"github.com/gravitational/teleport/lib/modules"
 	"github.com/gravitational/trace"
 )
 
 // GetAccessRequestMonthlyUsage returns the number of access requests that have been created this month.
-func GetAccessRequestMonthlyUsage(ctx context.Context, alog events.AuditLogger, now time.Time) (*proto.AccessRequestUsage, error) {
-	features := modules.GetModules().Features()
-	monthlyLimit := features.AccessRequests.MonthlyRequestLimit
-
+func GetAccessRequestMonthlyUsage(ctx context.Context, alog events.AuditLogger, now time.Time) (int, error) {
```

Comment:
I think the previous signature was the better one: `*proto.AccessRequestUsage` vs `int`.

That said, since we have a separate service, do we even need this method to exist (and be public)?

Reply:
At this point, this has been moved to `lib/resourceusage`, and I don't think it's necessarily worth tying it to the proto messages, as only the `resourceusagev1` service in `e` handles these proto contracts now.